lib/build.command.js

```diff
@@ -9,36 +9,35 @@
 module.exports = function (Aquifer) {
   'use strict';
 
+  // If the current directory doesn't contain an aquifer project, exit.
+  if (!Aquifer.initialized) {
+    return;
```

Comment:
@mattgrill This file, when included, adds a `build` command to the aquifer CLI. If Aquifer is not initialized (meaning the current folder is not an Aquifer project), then it returns before the commands are added.

Alternatively, I could load this module only upon deciding that Aquifer is initialized, in `bin/aquifer.js`. That's a better idea, I think, it at least makes the exclusion/inclusion of these modules more clear.